### PR TITLE
Feature/build api calls

### PR DESCRIPTION
**Pull request by masaki-kleinkopf**
What does this PR do?
-This builds out the fetch call for our get cocktail feature, and builds out our component for BoosterDrinkCard and adds conditional rendering. 
Where should your reviewer start?
-src/views/RandomView.vue
What (if any) features are you implementing?
- Building out the random cocktail booster functionality.
What (if anything) did you refactor?
-
Were there any issues that arose?
-Yes, working with the cocktailDB api.
Is there anything that you need from your teammate?
-
Any other comments, questions, or concerns?
-

### CODE REVIEW

File: src/views/RandomView.vue [hunk 1]
```diff
@@ -17,20 +17,63 @@ import { ref } from "vue"
...
+        preparation: data.drinks[0].strInstructions,
+        ingredients: ingredientsList,
+        id:Date.now(),
+        type:"drink",
+        isFavorited:false
+      }
+      return booster.value = newBooster;
+  })
+      // const newBooster = {
+      //   text: data.text,
```
Comment (Cyanne-Jones) [inline src/views/RandomView.vue:46]: This commented out section could be removed eventually on a future refactor branch but no rush.

File: src/views/RandomView.vue [hunk 2]
```diff
@@ -17,20 +17,63 @@ import { ref } from "vue"
...
+
+  const getNewDrink = () => {
+    fetch("https://www.thecocktaildb.com/api/json/v1/1/random.php")
+    .then(response => {
+      console.log(response)
+      return response.json()
+      })
+    .then(data => {
+      const ingredientsList = [];
+      for (let i = 1; i < 16; i++) {
```
Comment (davidhdaw) [inline src/views/RandomView.vue:29]: Going to tell my pebble that there's still a reason for for loops.

File: src/views/RandomView.vue [hunk 3]
```diff
@@ -17,20 +17,63 @@ import { ref } from "vue"
...
 <template>
   <div class="random">
     <div class="button-container">
       <button @click="getNewFact">Give me a new fact</button>
       <button>Give me a new song</button>
-      <button>Give me a new drink</button>
+      <button @click="getNewDrink">Give me a new drink</button>
     </div>
-    <BoosterCard v-if= "booster" 
+    <BoosterCard v-if= "booster && booster.type ==='fact'"
```
Comment (Cyanne-Jones) [inline src/views/RandomView.vue:63]: Let's change the name of this component eventually but it's not extremely urgent!

File: src/components/DrinkBoosterCard.vue [hunk 4]
```diff
@@ -0,0 +1,84 @@
...
+            <li v-for="ingredient in ingredients">
+                {{ingredient}}
+            </li>
+        </ul>
+        <p class="card-text">{{ preparation }}</p>
+    </div>
+</template>
+
+<style scoped>
+.booster-card {
```
Comment (davidhdaw) [inline src/components/DrinkBoosterCard.vue:30]: Can probably unsope this and have it work for all three card components?

**Review by davidhdaw — APPROVED** [review]

**Review by Cyanne-Jones — APPROVED** [review]
This api returned really strange data and this branch made the most of that gracefully.

**Pull request merged**